crates/ruma-events/src/room/member.rs

```diff
@@ -97,8 +100,7 @@ pub struct RoomMemberEventContent {
     #[cfg(feature = "unstable-pre-spec")]
     #[cfg_attr(docsrs, doc(cfg(feature = "unstable-pre-spec")))]
     #[serde(skip_serializing_if = "Option::is_none")]
-    #[ruma_event(skip_redaction)]
-    pub join_authorised_via_users_server: Option<UserId>,
+    pub join_authorised_via_users_server: Option<Box<UserId>>,
```

Comment:
Well after removing the FIXME I was just going to squash it all into 1 commit since the only other thing besides events/members.rs is Cargo.toml adding a needed feature

Reply:
Sounds good.